template-for-repository/proofs/Makefile.common

```diff
@@ -121,9 +121,19 @@ CBMCDIR ?= $(PROOF_ROOT)/cbmc
 # Default CBMC flags used for property checking and coverage checking
 CBMCFLAGS += --unwind 1 $(CBMC_UNWINDSET) --flush
 
+# Do property checking with the external SAT solver given by
+# EXTERNAL_SAT_SOLVER.  Do coverage checking with the default solver,
```

Comment:
Should we also document where this variable can be typically defined, something like:

```
Do property checking with the external SAT solver given by
EXTERNAL_SAT_SOLVER variable from the environment or Makefile-project-defines.
```

If this is not the best place to document it, please add it elsewhere, but a search for `EXTERNAL_SAT_SOLVER` on this repo currently returns only the `ifneq` line below and no documentation.

Reply:
Done.  Thanks!